### DISCUSSION TITLE
Welcome to ESPN_Extractor Discussions!

### DISCUSSION THREAD

**Discussion by adamvaldez** [adamvaldez/ESPN_Extractor#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lightsout7** [comment]
Hello, I have very limited experience in coding. But I want to learn Python and have the ability to write code for various hobbies or projects that I am working on at the time. Currently, I am trying to create a statistics analysis dashboard that updates daily or weekly for ESPN fantasy football and ESPN basketball. I was curious if this tool could be refitted to extract statistics for ESPN fantasy basketball as well.

**Reply by adamvaldez** [reply]
Hello @lightsout7, this tool is using [espn_api](https://github.com/cwendt94/espn-api) a "python package that ESPN's Fantasy API to extract data". You might be better off using that directly? Otherwise if you need the data in excel, csv, or other formats this tool might be what you want to use.

The [espn_api](https://github.com/cwendt94/espn-api) tool also supports basketball, baseball, and hockey. If there is demand, this tool could expand to support those sports as well.

Thanks for your comment!
